### PR TITLE
HW1 Submission

### PR DESCRIPTION
**Pull request by jerrashi**
Initially for this assignment, I tried using the Shell in R but it proved hard to use. Deb pointed me in the right direction towards RStudio and things got MUCH easier! After ever few edits to the readme, I staged the edits, then committed them, and finally pushed it to update the readme in my forked repository. After repeating this cycle until I reached my final draft of my submission, I created a pull request and attached this comment you're reading to submit the assignment.

To create the stylistic elements, I first copied the photo I wanted to use to the working directory. Then I inserted it with an exclamation point and hard brackets (where I could have put in a subtitle if I wanted to) and then the name of the file in soft brackets. To create headers, I used the r markdown syntax link and noticed that setext-style headers are “underlined” using equal signs (for first-level headers) and dashes (for second-level headers). I used the latter for the first header and the former for the second header. To create the lists, I used a single asterisk, then a numbered list. I had to make sure there was a new line before the start of the list, other my list was showing up as ( * item a * item b * item c). Similar to the photo, I inserted a link with the title in hard brackets and the link in soft brackets; no special character was needed. Text was italicized using a single asterisk (i.e. *text gets italicized text).

Looking forward to continuing to learn more computer science in this course! :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,31 @@
...
+
+Thai rolled ice cream
+
+3. Favorite genre of music?
+
+Pop all the way
+
+4. Fun fact?
+
+Chinese was my first language since I lived with my grandparents in China as a baby! Now English is the language I'm better at though, having grown up in the US.
```
Comment (whitingam) [inline README.md:31]: Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |       x    |              |            |
| **Coding strategy**         |      x     |              |            |
| **Presentation**            |    x       |              |            |
| **Achievement, creativity** |    x       |              |            |
| **Ease of access**          |      x     |              |            |

I think that this is an excellent piece of work!  I did not know how to do the subtitles that you used here, but now after seeing your example, I do.  I enjoyed reading your creative bio and appreciated your detailed comments about your process.

**Review by JiaxiLi1995 — CHANGES_REQUESTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      x     |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |     x      |              |            |
| **Achievement, creativity** |    x       |              |            |
| **Ease of access**          |     x      |              |            |

Remarks:
* Elaborate on above, especially for "needs work."
It is an interesting introduction.

* Some specific praise?
The whole assignment has a clear structure. You had used different headers and different kind of lists. It is excellent that you had a link for the photo.

* Something I learned?
It is great to learn from your project that setext-style headers are “underlined” using equal signs (for first-level headers) and dashes (for second-level headers). 

* Specific constructive criticism?
I think it would be better to name your pull request with your name on it. It might be easier to find.

* Something I know and that you, my peer, might like to know because it is relevant to something you struggled with.
I struggled with setting up the git in R too. Using shell is really confusion since it is like Command Prompt.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,31 @@
-# hw01
-Repository for homework 01
```
Comment (jfan3) [inline README.md:2]: Nice!

File: README.md [hunk 3]
```diff
@@ -1,2 +1,31 @@
...
+ * exploring downtown (which I *definitely* do not do enough!)
+ 
+A little more about me
+----------------------
+
+1. Hometown?
+
+I grew up all over the place! Born in Reno, then lived in SoCal (LA & San Diego) through middle school, then went to high school in Indiana.
+
+2. Favorite food?
```
Comment (jfan3) [inline README.md:21]: When writing enumerating list, you can write it in the code below, no need to spell out the numbered points.
```
1. point
    1. subpoint
    1. subpoint
```

**Review by jfan3 — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |x           |              |            |
| **Coding strategy**         |x           |              |            |
| **Presentation**            |            |x              |            |
| **Achievement, creativity** |           |x              |            |
| **Ease of access**          |x           |              |            |

Remarks:

* Nice work overall! 
* Can use more sections.

**Pull request closed without merging**